backwardcompatibilityml/tensorflow/models.py

```diff
@@ -5,6 +5,40 @@
 
 
 class BCNewErrorCompatibilityModel(tf.keras.models.Sequential):
+    """
+    BackwardCompatibility base model for Tensorflow
+
+    One may create a new Tensorflow model by subclassing
+    their new model h2 from this model.
+    This allows one to train or update a new model h2, using the
+    backward compatibility loss, with respect to an existing
+    model h1, using the Tensorflow fit method, h2.fit(...).
+
+    Assuming that you have a pre-trained model h1 and you
+    would like to create a new model h2 that you train
```

Comment:
Change to:  "would like to create a new model h2 trained"

Reply:
Done.